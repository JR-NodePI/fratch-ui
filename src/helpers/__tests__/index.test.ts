import { describe, expect, it } from 'vitest';

import * as index from '../index';

describe('index', () => {
  it('should have a list of index', () => {
    expect(index).toMatchSnapshot();
  });
});
