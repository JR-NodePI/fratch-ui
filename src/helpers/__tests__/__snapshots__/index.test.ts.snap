// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`index > should have a list of index 1`] = `
{
  "c": [Function],
  "getContrastColor": [Function],
  "getRandomColor": [Function],
  "hasClosestElement": [Function],
  "hexToRgb": [Function],
  "setStyleProperties": [Function],
}
`;
